src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -1672,6 +1672,86 @@ class C
         }
 
         [Fact]
+        public void CS0106ERR_BadMemberFlag05()
+        {
+            var text = @"
+struct Foo
+{
+    public abstract void Bar();
+}
+";
+            DiagnosticsUtils.VerifyErrorsAndGetCompilationWithMscorlib(text,
```

Comment:
Consider following style used by CS0106ERR_BadMemberFlag03 for the baseline. It is easier to read. If you run test with empty argument list for VerifyDiagnostics, the failure output for the unit-test will produce C# code for the argument list that you can simply copy/paste as the baseline.